### ISSUE TITLE
Workflow moderation emails are not sent

### ISSUE THREAD

**Issue by kapil-bunni** [openequella/openEQUELLA#4539]
**Describe the bug**
Not getting workflow moderation emails

--> I have given the SMTP details and able to send the test email.
--> Able to send search query emails and getting emails on resource rejection and when my resource has become live.
--> Not getting emails on workflow moderation steps.
--> I have given the **DISCOVER_ITEM** and **DIGITAL_RIGHTS_ITEM** privilege but it is of no use

**Expected behavior**
--> Should send workflow moderation emails on workflow moderation.

**Stacktrace**
No stack Trace everything works fine but emails are not sent.

**Platform:**

- OS: Ubuntu 20.04
- Browser chrome

Out Of Context:
I have also raised another [issue](https://github.com/openequella/openEQUELLA-ebi/issues/13) , please check.

Thank you.

**Comment by AlexPicone** [comment]
Hi, 
As this is a configuration issue as opposed to a coding issue, I'd recommend you email **info@edalex.com** to discuss how we can assist you with this issue.  Alternatively, other adopters of the platform may respond to this ticket.
Regards,
Alex

**Comment by kapil-bunni** [comment]
Hi, 

In openequella we have users, groups and roles under user management. I generally rely on roles for user segregation.

Coming to the exact issue : "moderation emails are not getting triggered on workflow moderation", I did some reaearch to solve the issue and on checking codebase for the solution I encountered workflow moderation emails are sent only for users and groups mapped to that particular workflow moderation step and dont'know why but for some reason roles are not included. 

There are two solutions to solve this :

1. Modify code in such a way that groups are also taken into consideration while triggering emails on workflow moderation.

2.Create a group with same members as role and map this group as moderators in workflow editor under moderators tab.

I prefer second solution.

Hope this helps and I'm closing the issue.

Thankyou.
